QC/wdl/tasks/deepvariant.wdl

```diff
@@ -0,0 +1,97 @@
+version 1.0
+
+workflow runDeepVariant {
+
+    call DeepVariant
+
+    output{
+        File deepVariantVCF    = DeepVariant.vcfOut
+        File deepVariantVCFTBI = DeepVariant.vcfIdxOut
+        File deepVariantHTML   = DeepVariant.visualReport
+    }
+}
+
+task DeepVariant{
+    input{
+        File inputReads
+        File inputReadsIdx
+        File assembly
+        File assemblyIndex
+        String sample
+
+        String modelType = "HYBRID_PACBIO_ILLUMINA"
+        String? callRegions
+
+        Int memSizeGB = 128
+        Int threadCount = 64
+        Int diskSizeGB = 128
+        String dockerImage = "google/deepvariant@sha256:440074e9cf854e20e3e05eab0a4fbbc32652c1f0d71b2aacbbd35da47f84faae" # 1.2.0
+
+
+    }
+
+    parameter_meta {
+        inputReads: "Reads aligned to assembly. must be in BAM or CRAM format."
+        inputReadsIdx: "Index file for BAM/CRAM."
+        assembly: "Assembly (reference) to that reads are aligned to. If compressed, must be compressed w/ bgzip."
+        assemblyIndex: "Index (fai) file for assembly"
+        modelType: "modelType type to use in variant calling. Should be one of: WGS,WES,PACBIO,HYBRID_PACBIO_ILLUMINA"
+    }
+
+
+    String outputVCF    = "~{sample}_deepvariant.vcf.gz"
+
+    command <<<
+        # exit when a command fails, fail with unset variables, print commands before execution
+        set -eux -o pipefail
+        set -o xtrace
```

Comment:
Always take my bash advice with a grain of salt, but according to [this](https://renenyffenegger.ch/notes/Linux/shell/bash/built-in/set/x), `set -o xtrace` is the same as `set -x` which you did in the line above, so this line is probably not needed.

Reply:
Good point.